lib/embed/gpi_embed.c

```diff
@@ -50,6 +50,22 @@ static char *argv[] = { progname };
 
 static PyObject *pEventFn = NULL;
 
+/**
+ * @name    Print the traceback
+ * @brief   Print traceback to stderr
+ *
+ * GILState before calling: N/A
+ *
+ * GILState after calling: N/A
+ *
+ * Print traceback and flush stderr if required (on Windows)
+ */
+void print_traceback(void) {
+  PyErr_Print();
+#if defined(_WIN32)
+  PyRun_SimpleString("import sys; sys.stderr.flush()");
```

Comment:
@AlexanderSpirin If the error occured **after** successful import of all modules and there is no syntax errors, then errors visible both in Modelsim and console and log file. Syntax errors displayed into console and they still visible. But before correct import of that test unit - simply "failed to import module" and nothing more.